src/test/java/org/kie/processmigration/service/PlanServiceImplTest.java

```diff
@@ -43,7 +42,7 @@
     @Inject
     PlanService planService;
 
-    @AfterEach
+    @BeforeEach
```

Comment:
Why is this needed? I'd clean up something after I've done changes

Reply:
Originally the problem was that flyway didn't clean up the database, so there might be some leftover from previous tests. I now forced flyway to clean the db at startup for the tests, so it should be ok now.